src/ProxyClient/Invalidation/TagsInterface.php

```diff
@@ -29,13 +29,11 @@
     public function invalidateTags(array $tags);
 
     /**
-     * Get the value for the HTTP tag header.
-     *
-     * This concatenates all tags and ensures correct encoding.
+     * Get escaped tags
      *
      * @param array $tags
      *
-     * @return string
+     * @return array
      */
     public function getTagsHeaderValue(array $tags);
```

Comment:
if we return a tag array, what would this method do at all? can you check how escaping is handled? i guess one thing that a PSR-7 library won't care about is `,` in a header - but if there are any `,` in a tag, we need to escape them.